docs/changelog/138492.yaml

```diff
@@ -0,0 +1,61 @@
+pr: 138492
+summary: Enable bfloat16 and on-disk rescoring for dense vectors
+area: Vector Search
+type: feature
+issues: []
+highlight:
+  title: New dense_vector options for storing bfloat16 vectors and utilising on-disk
+    rescoring
+  body: |-
+    New options have been added to the `dense_vector` field type.
+
+    The first is support for storing vectors in https://en.wikipedia.org/wiki/Bfloat16_floating-point_format[bfloat16]
+    format. This is a floating-point format that utilises 2-bytes per value rather than 4, halving the storage space
+    required compared to `element_type: float`. This can be specified with `element_type: bfloat16`
+    when creating the index, for all `dense_vector` indexing types:
+
+    [source,yaml]
```

Comment:
FYI this is asciidoc syntax

Reply:
I've removed a few of the examples, trimming it down a bit